assets/js/edit.js

```diff
@@ -20,6 +20,9 @@ const { Fragment } = wp.element;
 const { apiFetch } = wp;
 const ALLOWED_MEDIA_TYPES = [ 'audio' ];
 
+// Import taxonomy panel.
+import PostTaxonomies from './taxonomy-panel';
```

Comment:
Nit: I'd give the component and file the same name: either `PostTaxonomies from './post-taxonomies'` or `TaxonomyPanel from ...`